bit/network/services.py

```diff
@@ -11,6 +14,104 @@ def set_service_timeout(seconds):
     DEFAULT_TIMEOUT = seconds
 
 
+class RPCHost:
+    def __init__(self, user, password, host, port, use_https):
+        self._session = requests.Session()
+        self._url = "http{s}://{user}:{password}@{host}:{port}/".format(
+            s="s" if use_https else "",
+            user=user,
+            password=password,
+            host=host,
+            port=port,
+        )
+        self._headers = {"content-type": "application/json"}
+        self._session.verify = use_https
+
+    def __getattr__(self, rpc_method):
+        return RPCMethod(rpc_method, self)
+
+    def get_balance(self, address):
+        getcontext().prec = 9
+        b = Decimal(self.getreceivedbyaddress(address, 0))
+        return int(float(str(b * 100000000)))
```

Comment:
Don't we already have constants in place for this?

Reply:
Ah, you are right. They are inside [`network/rates.py`](https://github.com/ofek/bit/blob/0b0e553d8627160c624f8a9d811e4596338b4689/bit/network/rates.py#L20). I will change it to make use of them.